plugins/source_testing.go

```diff
@@ -107,8 +116,15 @@ func validateResources(t *testing.T, resources []*schema.Resource) {
 
 	// Make sure every column has at least one value.
 	for i, hasValue := range columnsWithValues {
-		if !hasValue && !(table.Columns[i].Name == "_cq_parent_id" && table.Parent == nil) {
-			t.Errorf("table: %s column %s has no values", table.Name, table.Columns[i].Name)
+		col := table.Columns[i]
+		switch {
+		case col.IgnoreInTests,
+			hasValue,
+			col.Name == "_cq_parent_id" && table.Parent != nil:
+		// nop
+		default:
+			errs = append(errs, fmt.Errorf("table: %s column %s has no values", table.Name, table.Columns[i].Name))
 		}
```

Comment:
```suggestion
		emptyExpected := col.Name == "_cq_parent_id" && table.Parent == nil
		if !col.IgnoreInTests && !emptyExpected && !hasValue {
		errs = append(errs, fmt.Errorf("table: %s column %s has no values", table.Name, table.Columns[i].Name))
		}
```

Maybe this ☝️  (I think this also fixes a bug related to the parent columns being null?)